### DISCUSSION TITLE
[PSD7] Esercizio Slide 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#152] [Esercizi (A-L)]
Semplificare la seguente equazione booleana:

![image](https://user-images.githubusercontent.com/69389461/210017466-85789b0d-f8e4-48f0-bd62-68e136f6022c.png)

**Comment by Elia-Belli** [comment]
$d (ac+cb) + d(\overline{\bar a c + a \bar c})$
$d (ac+cb + (\overline {\bar a c}) (\overline{a \bar c}))$
$d (ac+cb + (a+ \bar c) (\bar a + c))$
$d (ac+cb + ac + \bar c \bar a)$
$d (ac + cb + \bar c \bar a)$
$acd + bcd + d\bar a \bar c$

**Comment by fraxb04** [comment]
(!(!A)C)(!(A!(C))) = AC +!A!C
quindi il risultato è diverso

**Comment by Elia-Belli** [comment]
@fraxb04 ho corretto.
Cmq l'errore era alla terzultima riga, avevo scritto $\bar c a$ anzichè $\bar c \bar a$, adesso torna
